PowerArubaCX/Public/Radius_group.ps1

```diff
@@ -0,0 +1,244 @@
+#
+# Copyright 2021, Cédric Moreau <moreaucedric0 at gmail dot com>
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+function Add-ArubaCXRadiusServerGroup {
+
+    <#
+        .SYNOPSIS
+        Add Aruba CX RADIUS Server Group
+
+        .DESCRIPTION
+        Add RADIUS Server Group (group_name, group_type)
+
+        .EXAMPLE
+        Add-ArubaCXRadiusServerGroup -group_name PowerArubaCX
+
+        Add RADIUS Server Group with name PowerArubaCX
+
+        .EXAMPLE
+        Add-ArubaCXRadiusServerGroup -group_name PowerArubaCX -group_type radius
+
+        Add RADIUS Server Group with name PowerArubaCX and group type radius
+    #>
+
+    Param(
+        [Parameter (Mandatory = $true)]
+        [string]$group_name,
+        [Parameter (Mandatory = $false)]
+        [ValidateSet('radius','tacacs')]
+        [string]$group_type = "tacacs",
```

Comment:
Why Tacacs by default ?

Rename function to CXAAAServerGroup ? (like AAA call ?)